test/controllers/v1/profiles_controller_test.rb

```diff
@@ -11,6 +11,29 @@ class ProfilesControllerTest < ActionDispatch::IntegrationTest
       profiles(:one).update! account: accounts(:test)
     end
 
+    context 'index' do
```

Comment:
Somehow we overlooked index controller tests when the PR went in. I'd specifically like a test that shows that a user can still query for all internal and external, canonical and non-canonical profiles (essentially all profiles that are not a part of someone else's account) with this change.